client/dashboard/profile-wizard/index.js

```diff
@@ -90,32 +88,26 @@ class ProfileWizard extends Component {
 		return currentStep;
 	}
 
-	goToNextStep() {
+	async goToNextStep() {
+		const { addNotice, isError, updateProfileItems } = this.props;
 		const currentStep = this.getCurrentStep();
 		const currentStepIndex = getSteps().findIndex( s => s.key === currentStep.key );
 		const nextStep = getSteps()[ currentStepIndex + 1 ];
 
 		if ( 'undefined' === nextStep ) {
-			this.updateProfile( { complete: true } );
+			await updateProfileItems( { complete: true } );
```

Comment:
I tried commenting out the other steps here, so that clicking the get started button mark the profiler as completed, but I'm getting the following:

`Uncaught (in promise) TypeError: Cannot read property 'apply' of undefined`.

Reply:
Thanks for noting that!  Turns out I had written `dispatch( 'wc-admin' )` below so this wasn't really connected to `wc-api`.